QEfficient/generation/text_generation_inference.py

```diff
@@ -237,7 +240,7 @@ def cloud_ai_100_exec_kv_helper(
     print()
 
     latency_stats = (generated_texts, prefill_perf, decode_perf, total_perf, total_time)
-    return latency_stats
+    return latency_stats, generated_ids
```

Comment:
Create a `dataclass` that holds all the info about `Cloud AI 100 execution` and return it from here.
Also, make `cloud_ai_100_exec_kv` return the same `dataclass` object.

```
Class CloudAI100ExecInfo
    generated_texts: str
    generated_ids: List[int]
    prefil_perf: float
    decode_perf: float
    ...
```

Reply:
Updated